_search-plugins/star-tree-index.md

```diff
@@ -208,38 +208,35 @@ To use date histogram aggregations and make them searchable in a star-tree index
 - The calendar intervals in a star-tree mapping configuration can use either the request's calendar field or a field of lower granularity than the request field. For example, if an aggregation uses the `month` field, the star-tree search can still use lower-granularity fields such as `day`.
 - A metric sub-aggregation must be part of the aggregation request.
 
-The following example gets the sum of all the values in the `size` field with a range query, aggregated for each calendar month, for all error logs containing `method:get`:
+The following example gets the sum of all the values in the `size` field with a range query, aggregated for each calendar month:
```

Comment:
@kolchfa-aws: how does this sound?

```suggestion
The following example gets a range of status codes between `200` and `400`, then aggregates the sum of all the values for each calendar month:
```

Reply:
This is good. What does the `size` field contain? Is it the request size? Response size? It would be nice to specify this. In terms of the intro sentence, how about:
```suggestion
The following example first filters logs to include only those with status codes between `200` and `400`. It then aggregates the filtered logs by calendar month and calculates the total sum of values in the `size` field for each month:
```

Reply:
@kolchfa-aws: Adjusted.

Reply:
Thank you but could you clarify what is meant to be kept in the `size` field? I think it's essential for users' understanding of this example.

Reply:
`size: 0` indicates that the response will not return individual document hits and instead only return the aggregation results. Is that what you're referring to, or are you referring to the `"field": "size"` in the aggregation itself?